<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="MachineHuman">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine vs. Human</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HumanMachine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Human vs. Machine</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MachineMachine">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Machine vs. Machine</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HumanHuman">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Human vs .Human</string>
    </property>
   </widget>
   <widget class="QLabel" name="P1P2label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>      P1          P2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartOver">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Over</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
